src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -306,14 +323,12 @@ public async Task SynchronizeWithServer_ConnectedModeWithIssues_UIContextIsSetAn
                 toolWindowService, serviceProvider, logger);
         }
 
-        private static Mock<ISonarQubeService> CreateSonarServerMock(bool isServerConnected, params SonarQubeIssue[] serverIssues)
```

Comment:
I decided to remove this method and instead use the explicit `CreateSonarQubeService`, `SetupTaintIssues`, `SetupAnalysisInformation`.
The method would have had another optional parameter (analysis information) and it just became too complicated to read.